### PR TITLE
Add FirstBlood to existing dapps

### PR DESCRIPTION
**Pull request by auryn-macmillan**

### CODE REVIEW

**Comment by alwx** [comment]
@auryn-macmillan thank you for your PR! Everything is fine, except one small tabulation issue. Of course, it is not a big thing, but it's better to change it to make all spacings equal.

After doing it, please, squash all your commits into one.

File: resources/default_contacts.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
         {
             "en": "http://auctionhouse.dappbench.com"
         }
-    },
+   },
```
Comment (alwx) [inline resources/default_contacts.json]: Please, change it back.

Comment (auryn-macmillan) [inline resources/default_contacts.json]: ok, I've fixed the tabulation error. The commits can be squashed when the pull request is merged, correct?

**Comment by auryn-macmillan** [comment]
I fixed the tabulation error. You should be able to squash the commits when the pull request is merged, correct?

**Comment by alwx** [comment]
@auryn-macmillan sure, I've completely forgotten that Github introduced this feature not so long ago :)

Thank you!

**Pull request merged**